docs/learn/advanced/08-events.md

```diff
@@ -57,7 +57,7 @@ The following examples show how to query Events using the Cosmos SDK.
 | `tx.height=23`                                   | Query all transactions at height 23                                                                                                                      |
 | `message.action='/cosmos.bank.v1beta1.Msg/Send'` | Query all transactions containing a x/bank `Send` [Service `Msg`](../../build/building-modules/03-msg-services.md). Note the `'`s around the value.                  |
 | `message.module='bank'`                          | Query all transactions containing messages from the x/bank module. Note the `'`s around the value.                                                       |
-| `create_validator.validator='cosmosval1...'`     | x/staking-specific Event, see [x/staking SPEC](../../build/modules/staking/README.md).                                                         |
+| `create_validator.validator='cosmosval1...'`     | x/staking-specific Event, see [x/staking SPEC](../../../x/staking/README.md).                                                         |
```

Comment:
I don't think docusaurus will be able to resolve this when this will get copied to the doc repo.
I think the link is actually correct when built in the doc repo, but incorrect here, so maybe we should skip?